testcase/resolve-lock/resolve_lock.go

```diff
@@ -568,7 +568,7 @@ func (c *resolveLockClient) getTs(ctx context.Context) (uint64, error) {
 			ts := oracle.ComposeTS(physical, logical)
 			return ts, nil
 
-		case io.EOF:
+		case io.EOF, errors.New("context canceled"):
```

Comment:
```suggestion
		case io.EOF, context.Canceled:
```

`context.Canceled` is `errors.New("context canceled")` though. https://github.com/golang/go/blob/dc65c489cc5a795a68d844ed7a45e5d16562401d/src/context/context.go#L157